pypeit/core/skysub.py

```diff
@@ -1041,7 +1041,8 @@ def local_skysub_extract(sciimg, sciivar, tilts, waveimg, global_sky, thismask,
         canvas_list = points_mask + points_omask + text_mask + text_omask
         canvas.add('constructedcanvas', canvas_list)
 
-    return skyimage[thismask], nobkg_skyimage[thismask], objimage[thismask], modelivar[thismask], outmask[thismask]
+    return skyimage[thismask], nobkg_skyimage[thismask] if nobkg_skyimage is not None else None, \
```

Comment:
can you document why `nobkg_skyimage` may be `None`?

Maybe add it to the `Returns` portion of the doc string